README.md

```diff
@@ -36,7 +36,8 @@ export MAPBOX_ACCESS_TOKEN=my.token
 # Commands
 
 * Tileset Sources
-  * [`add-source`](#add-source)
+  * [`upload-source`](#add-source)
```

Comment:
Anchor should point to upload-source

Reply:
Why am I so weirdly bad at updating anchor links?